### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Zelalem1222**
In this PR 

- I added styling that matches Zeplin's design.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,17 +1,142 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem 1rem;
+  height: 50px;
+  font-family: var(--montserrat);
+  @media screen and (min-width: 768px) {
+    height: 80px;
+  }
+`;
```
Comment (roshan-bajgain) [inline src/components/Navbar.js]: - [ ] I would suggest you stick as much as possible to the Figma design. Book store CMC and logo at the right should be aligned with the added book. See the image below:
### Your design
your
### Figma design
figma

File: src/components/book.js [hunk 2]
```diff
@@ -1,38 +1,266 @@
...
+          {isDesktop && (
+            <ProgressNumContainer>
+              <ProgressStatistics>
+                {Progress}
+                %
+              </ProgressStatistics>
+              <ProgressCompleted>Completed</ProgressCompleted>
+            </ProgressNumContainer>
+          )}
+        </BookProgressContainer>
```
Comment (roshan-bajgain) [inline src/components/book.js]: - [ ] I would suggest you make your design similar to the Figma design as it is the requirement. Kindly make changes to the progress bar.
### your design
your
### Figma design
figma

File: src/styles/addBooks.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
+form {
+  width: 90%;
+  margin-inline: auto;
+  margin-bottom: 30px;
+}
```
Comment (roshan-bajgain) [inline src/styles/addBooks.css:5]: - [ ] When I notice your form then it is not covering the whole width but if you see the Figma design then the input form is covering the maximum length. Kindly adjust it.
- [ ] kindly change the color of `Add buttons` as well
### Your design
your
### Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Zelalem1222, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi @Zelalem1222 ,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**